projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/forgot-password.a11y-e2e.cy.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+describe('Forgot Password Page Accessibility', () => {
+  beforeEach(() => {
+    cy.a11yContinuumSetup();
+    cy.visit('/login/forgot-password');
+    cy.get('cx-forgot-password').should('exist');
```

Comment:
```suggestion
    cy.get('cx-forgot-password')
```

We should not use assertions. Ideally the only failure point should come from Access Continuum itself.

Reply:
I've just updated.